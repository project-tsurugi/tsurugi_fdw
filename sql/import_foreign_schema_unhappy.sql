/* Test case: 1-4 */
-- Test
IMPORT FOREIGN SCHEMA public FROM SERVER tsurugidb INTO public OPTIONS (name 'value');

/* Test case: --- */
-- Test
IMPORT FOREIGN SCHEMA public FROM SERVER tsurugidb INTO unknown_schema;

/* Test case: 5-1 */
-- Test
IMPORT FOREIGN SCHEMA public FROM SERVER unknown_server INTO public;
