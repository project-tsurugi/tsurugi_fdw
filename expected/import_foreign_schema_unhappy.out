/* Test case: 1-4 */
-- Test
IMPORT FOREIGN SCHEMA public FROM SERVER tsurugidb INTO public OPTIONS (name 'value');
ERROR:  unsupported import foreign schema option "name"
/* Test case: --- */
-- Test
IMPORT FOREIGN SCHEMA public FROM SERVER tsurugidb INTO unknown_schema;
ERROR:  schema "unknown_schema" does not exist
/* Test case: 5-1 */
-- Test
IMPORT FOREIGN SCHEMA public FROM SERVER unknown_server INTO public;
ERROR:  server "unknown_server" does not exist
